src/pymor/vectorarrays/block.py

```diff
@@ -178,8 +189,9 @@ def __eq__(self, other):
     def dim(self):
         return sum(subspace.dim for subspace in self.subspaces)
 
-    def zeros(self, count=1, reserve=0):
-        return BlockVectorArray([subspace.zeros(count=count, reserve=reserve) for subspace in self.subspaces], self)
+    def zeros(self, count=1, reserve=0, dtype=None):
+        return BlockVectorArray([subspace.zeros(count=count, reserve=reserve, dtype=None)
+                                 for subspace in self.subspaces], self)
```

Comment:
Yes, sure. In `sys-mor` branch, `algorithms.sylvester` creates an empty vectorarray from `A.source`. Operator `A` can be a `BlockOperator`, e.g. when transforming a second order system into a first order system (see `discretizations.iosys.SecondOrderSystem.to_lti`).